### ISSUE TITLE
Unproved VC with quantifier instance

### ISSUE THREAD

**Issue by clairedross** [cvc5/cvc5#10714]
Hi all,

**Describe the bug**
[example_smt2.txt](https://github.com/cvc5/cvc5/files/15186118/example_smt2.txt)
On the attached example, cvc5 returns unknown even though the last assertion is the negation of an instance of the universal quantifier just above. Adding or removing the trigger does not seem to change anything.

*Note*: If the bug is triggered with an SMT2 input file please minimize it with [ddsmt](https://github.com/ddsmt/ddSMT).

**Command line arguments**: cvc5  example.smt2
**cvc5 version/commit**: cvc5 1.1.2
**Operating system**: Linux

I understand that quantifier instantiation is a heuristic and in general best effort. If possible, I would be interested in advice on what to avoid for E-Matching to work with cvc5. Removing or adding a trigger does not seem to change anything. I also tried inlining the let bindings and removing the definition of aa but without effects.

Thanks in advance

**Comment by ajreynol** [comment]
Thanks a lot for the benchmark, it was helpful to see.

The reason for the failure is that cvc5 "miniscopes" quantified formulas independent of whether they have user patterns. This led to complications for inferring user patterns for *all* miniscoped conjuncts.

I believe we should preserve user patterns by default instead of miniscoping. I've opened a PR for this change to our defaults (https://github.com/cvc5/cvc5/pull/10719).

For reference, https://github.com/cvc5/cvc5/issues/6996 was an issue where the opposite policy was required (i.e. you must miniscope to solve the problem).  I've refined the behavior to incorporate this case as well.
